README.adoc

```diff
@@ -261,7 +261,7 @@ ENTRYPOINT ["java","-cp","app:app/lib/*","hello.Application"]
 
 There are now 3 layers, with all the application resources in the later 2 layers. If the application dependencies don't change, then the first layer (from `BOOT-INF/lib`) will not change, so the build will be faster, and so will the startup of the container at runtime as long as the base layers are already cached.
 
-NOTE: We used a hard-coded main application class `hello.Application`. This will probably be different for your application. You could parameterize it with another `ARG` if you wanted. You could also copy the Spring Boot fat `JarLauncher` into the image and use it to run the app - it would work and you wouldn't need to specify the main class, but it would be a bit slower on startup.
+NOTE: We used a hard-coded main application class `hello.Application`. This will probably be different for your application. You could parameterize it with another `ARG` if you wanted. You could also copy the Spring Boot fat `JarLauncher` into the image and use it to run the app -- it would work and you wouldn't need to specify the main class, but it would be a bit slower on startup.
```

Comment:
We use two hyphens here so that Asciidoctor will create an em dash for us. So this is not a change we would want to make.